Snippets/Core/Core_6/Scanning/ScanningPublicApi.cs

```diff
@@ -44,5 +52,28 @@ void ScanningUpgrade(EndpointConfiguration endpointConfiguration)
             #endregion
         }
 
+        void ScanningApDomainAssemblies(EndpointConfiguration endpointConfiguration)
+        {
+            #region ScanningApDomainAssemblies
+
+            endpointConfiguration.AssemblyScanner(scannerConfiguration =>
+            {
+                scannerConfiguration.ScanAppDomainAssemblies = true;
+            });
+
+            #endregion
+        }
+
+        void SwallowScanningExceptions(EndpointConfiguration endpointConfiguration)
+        {
+            #region SwallowScanningExceptions
+
+            endpointConfiguration.AssemblyScanner(scannerConfiguration =>
+            {
+                scannerConfiguration.ThrowExceptions = false;
```

Comment:
@timbussmann @andreasohlund isnt this against the new coding guidelines for the core?